kubernetes/iac/modules/hw-tf-module-cluster-resources/main.tf

```diff
@@ -0,0 +1,102 @@
+resource "helm_release" "dapr" {
```

Comment:
Can we split this into multiple files? One for each release. I don't think we can keep extending the one .tf file especially when we want to migrate all the helmfile configs to terraform.